xapian-core/tests/harness/backendmanager_remotetcp.cc

```diff
@@ -341,6 +348,9 @@ launch_xapian_tcpsrv(const string & args)
 # error Neither HAVE_FORK nor __WIN32__ is defined
 #endif
 
+// map exact database object to it's pid_fd position in pid_to_fd
+static std::map<std::string, unsigned> uuid_to_pididx;
```

Comment:
We don't want to use a `std::map` here - it causes problems because it dynamically allocates memory, and that confuses the test harness leak checking machinery as the existing comment above (~line 73) notes.

Just use a static fixed size array and linearly scan like we already do.  It doesn't need a lot of entries, so it's probably as cheap to linearly scan anyway (it being O(n) is irrelevant when n is small and fixed).